xbmc/music/Album.h

```diff
@@ -71,6 +71,12 @@ class CAlbum
   }
 
   std::string GetArtistString() const;
+  /*! \brief Get album artist names as a string vector from the vector of artist credits
```

Comment:
Sorry, I was this unspecific here.

Go for something along these lines for each brief:
```
/*! \brief Get album artist names from the artistcredits object
\return Album artist names as a vector
 */
````

There is also a ```\param``` line start, when you want to define parameters, not needed here. But just for future reference.